db/log_test.cc

```diff
@@ -979,6 +979,38 @@ TEST_P(CompressionLogTest, Fragmentation) {
   ASSERT_EQ("EOF", Read());
 }
 
+TEST_P(CompressionLogTest, AlignedFragmentation) {
+  CompressionType compression_type = std::get<2>(GetParam());
+  if (!StreamingCompressionTypeSupported(compression_type)) {
+    ROCKSDB_GTEST_SKIP("Test requires support for compression type");
```

Comment:
nit: also print out compression type `CompressionTypeToString`?

Reply:
Its a little cumbersome as ROCKSDB_GTEST_SKIP() doesn't take a format string. The compression type can be inferred from the full test name anyway.